docs/source/subgraphs.mdx

```diff
@@ -156,6 +156,85 @@ server.listen(4001).then(({ url }) => {
 });
 ```
 
+## Securing your subgraphs
+
+Because of the power and flexibility of the [`Query._entities` and `Query._service` fields](#subgraph-specific-symbols), your subgraphs should **not** be directly accessible by clients. Instead, only your [gateway](./gateway/) should have access to your subgraphs. Clients then communicate with the gateway:
+
+<FederationArchitecture />
+
+Make sure to implement any necessary firewall rules, access control
+lists, or other measures to ensure that individual subgraphs can
+be accessed only via the gateway.
+
+## Subgraph-specific symbols
+
+When you [generate your subgraph schema](#generating-a-subgraph-schema), some federation-specific definitions are automatically added to it. In addition to directive definitions like `@key`, the most useful of these definitions for debugging are two fields of the `Query` type: `_service` and `_entities`:
+
+```graphql
+type Query {
+  # ...your field definitions...
+
+  # Added automatically
+  _service: _Service!
+  _entities(representations: [_Any!]!): [_Entity]!
```

Comment:
Should we also briefly explain that the `_Entity` type is an automatically added `union` of all entity types?

```graphql
union _Entity = Review | User | Product # ... more!
```